product_bom_revision/i18n/product_bom_revision.pot

```diff
@@ -0,0 +1,33 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* product_bom_revision
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```

Comment:
I'm not sure what this .pot file is but it is showing the old version and probably should read 13.

Reply:
I don't know but there are other PR with the same file see https://github.com/OCA/product-attribute/pull/541 and the file was update so now it show "Project-Id-Version: Odoo Server 13.0\n" as you can see here product_multi_category/i18n/product_multi_category.pot

Reply:
Maybe it's auto updated then and it doesn't matter.